README.md

```diff
@@ -1,5 +1,9 @@
+[![Build Status](https://travis-ci.org/PenneyGadget/lookingfor.svg?branch=master)](https://travis-ci.org/PenneyGadget/lookingfor)
```

Comment:
Can you give me some more granular instructions on setting up? I've created the Travis account - in the meantime, I'll add you back as an Admin!